core/src/bson_type_info.rs

```diff
@@ -218,6 +218,52 @@ impl BsonTypeInfo {
         column_size: make_default_attr_func!(Some(MAX_STRING_SIZE)),
         simple_type_info: None,
     };
+    pub const WLONGVARCHAR: BsonTypeInfo = BsonTypeInfo {
+        type_name: "string",
+        sql_type: SqlDataType::SQL_WLONGVARCHAR,
+        non_concise_type: None,
+        searchable: SQL_SEARCHABLE,
+        is_case_sensitive: true,
+        fixed_prec_scale: false,
+        scale: None,
+        length: make_default_attr_func!(Some(MAX_STRING_SIZE)),
+        precision: None,
+        char_octet_length: make_default_attr_func!(Some(MAX_STRING_SIZE)),
+        transfer_octet_length: None,
+        display_size: make_default_attr_func!(Some(MAX_STRING_SIZE)),
+        literal_prefix: Some("'"),
+        literal_suffix: Some("'"),
+        sql_code: None,
+        is_auto_unique_value: None,
+        is_unsigned: None,
+        num_prec_radix: None,
+        decimal_digit: None,
+        column_size: make_default_attr_func!(Some(MAX_STRING_SIZE)),
+        simple_type_info: None,
+    };
+    pub const WCHAR: BsonTypeInfo = BsonTypeInfo {
```

Comment:
Can you add a similar comment than the one on top of "VARCHAR" explaining that this was added for PowerBI and is necessary because of the way the apply a cast using the type name.